desktop/app/window-handlers/notifications/index.js

```diff
@@ -41,15 +43,16 @@ function updateNotificationBadge( count ) {
 module.exports = function ( mainWindow ) {
 	ipc.on( 'unread-notices-count', function ( _, count ) {
```

Comment:
@belcherj Because there will be a potential (pretty much guaranteed) mismatch between the badge count derived "in realtime" from the websocket and Calypso's unread count, I think we want to disable the  existing `unread-notices-count` IPC event. However, we probably do want to retain the "clear badge count when notifications panel opened" behavior.

Reply:
removed.